### PR TITLE
fix(components): [dropdown] fix focus srcoll

### PR DESCRIPTION
**Pull request by gjfei**
当有聚焦元素在页面底部时，dropdown 聚焦时会滚动到底部

Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by tolking** [comment]
Hi, can you provide some demos?

**Comment by gjfei** [comment]
> Hi, can you provide some demos?

https://github.com/gjfei/test-dorpdown
先让底部元素聚焦，然后鼠标移动到dropdown 会让dropdown聚焦，页面就会发生滚动

**Comment by gjfei** [comment]
@tolking

File: packages/components/dropdown/src/dropdown.vue [hunk 1]
```diff
@@ -191,10 +191,10 @@ export default defineComponent({
       }
     }
 
-    function handleShowTooltip(event?: Event) {
-      if (event?.type === 'keydown') {
```
Comment (chenxch) [inline packages/components/dropdown/src/dropdown.vue:195]: Why is the judgment of `keydown` canceled here, do you only need to add `preventScroll: true`?

Comment (gjfei) [reply packages/components/dropdown/src/dropdown.vue:195]: > Why is the judgment of `keydown` canceled here, do you only need to add `preventScroll: true`?

我并不清楚当时为什么要添加keydown 的判断

Comment (gjfei) [reply packages/components/dropdown/src/dropdown.vue:195]: > 

嗯，我发现这样并没有效果，我昨天重新调试了一下，找不到为什么，你可以帮忙看下？

**Comment by xiaoxian521** [comment]
Hey, thank you very much for your contribution, but here are some conflicts that need to be solved by you. We can continue to review the code after solving. One thing to remind is that if the conflicts is still not resolved after `10` days, sorry, we will close this pr ❤️

**Pull request closed without merging**